SoSe19/bericht_data.tex

```diff
@@ -0,0 +1,80 @@
+Schwerpunkt in der Diskussion um Open Science waren die Themen Plan S \footnote{\url{https://www.coalition-s.org/about/}}, Project DEAL \footnote{\url{https://www.projekt-deal.de/}} und wie Open Science in der universitären Lehre verankert werden könnte.
+
+\section*{Bachelor-Master-Umfrage}
+
+Im Sommersemester 2018 wurde durch die ZaPF eine Umfrage unter den Physikstudierenden im deutschsprachigen Raum durchgeführt, welche zum Ziel hat, die Entwicklung des Physikstudiums in regelmäßigen Abständen zu beobachten. 
+Die Umfrage wurde zum dritten Mal durchgeführt und es nahmen 2750 Studierende aus dem Bachelor, Master oder Promotion daran teil. Es wurden hierbei allgemeine Fragen zu Bereichen wie Studienbelastung, subjektiver Wahrnehmung der Vermittlung von Qualifikationen und Kompetenzen, Vorbereitung auf das Studium sowie der Weg nach dem Studium gestellt. Darüber hinaus wurde analysiert, inwieweit die Themenkomplexe Ethik und Wissenschaftstheorie im Studium eingebunden werden sollten.
+
+Auf der ZaPF in Bonn hat sich ein Arbeitskreis darüber ausgetauscht, wie die Ergebnisse der Umfrage den Universitäten aufbereitet zur Verfügung gestellt werden können und an welchen Stellen sie außerdem eingesetzt werden können.
+
+\section*{Qualifikationsziele physikalischer Praktika}
+
+Praktika sind eine zentrale Lehrveranstaltungsform in naturwissenschaftlichen Fächern. Daher wurde sich bereits auf vergangenen ZaPFen mit Qualifikationszielen und Rahmenbedingungen physikalischer Anfangs- und Fortgeschrittenenpraktika beschäftigt. 
+Die Sommer-ZaPF 2019 hat ein Positionspapier zur Verbesserung von Fortgeschrittenenpraktika in der Physik verabschiedet. 
+Das Ziel dieser Fortgeschrittenenpraktika ist die Vermittlung von spezifischen inhaltlichen sowie formellen Qualifikationszielen und Schlüsselqualifikationen. Diese Ziele sind zentrale Fähigkeiten Studierender der Physik und sollen dabei im Fortgeschrittenenpraktikum erlernt und ausgebaut werden. Nach erfolgreichem Abschluss sollen Qualifikationen wie vertiefte Statistik- und Plotkenntnisse, ein nachhaltiges Forschungsdatenmanagement sowie ein übersichtliches Dokumentieren und Auswerten der gemessenen Daten vermittelt worden sein. Übergeordnetes Ziel ist dabei, eine Intuition für physikalische Zusammenhänge zu entwickeln. Die Gestaltung und Vermittlung dieser Qualifikationsziele obliegt dabei der Hochschule. 
+
+\section*{Programmierkentnisse}
+
+Die Vermittlung von Programmierkenntnissen ist ein vitaler Teil des Physikstudiums. Seit einiger Zeit beschäftigt sich die ZaPF daher damit, wie diese idealerweise im Studium integriert sein sollte. Auf der letzten ZaPF  In Bonn wurde deswegen hierzu ein Leitfaden \footnote{\url{https://zapf.wiki/Leitfaden_Programmierkentnisse_im_Physikstudium}} erstellt.
```

Comment:
Der erste Absatz der verlinkten Wiki-Seite sagt, dass das kein Leitfaden, sondern nur der Entwurf zu einem ist. Wollen wir wirklich auf den Entwurf von etwas in ZaPF-Bericht verlinken?

Außerdem ist das ein Link ins Wiki. Da kann sich also sehr schnell sowohl der Inhalt ändern, als auch die Seite verschoben werden und der Link dann ins leere führen. Vielleicht sollte hier gewartet werden bis evtl in Freiburg ein Leitfaden verabschiedet wird und man dann den Beschluss verlinken kann.